src/extension.ts

```diff
@@ -41,7 +45,7 @@ export async function activate(context: vscode.ExtensionContext) {
     }
 }
 
-async function getClusterKubeconfig(target: AksClusterTreeItem): Promise<string | undefined> {
+export async function getClusterKubeconfig(target: AksClusterTreeItem): Promise<string | undefined> {
```

Comment:
You don't actually need to do this I think - I think it is already exposed on the tree cluster object as `getKubeconfigYaml` or something like that.

Reply:
💡 Cool,  (Setting this note for myself as a context) I have tried but something seems disconnect, so I might show it to you once I have pushed my local changes. It is to do with `getKubeConfig` available at run time at `cloudExplorer.api.explorer.provider[0].getKubeConfig` but I am not sure how can I translate this to what we have.

Reply:
💡 Thank you for the discussion, it's implemented as external help function now. 🙏😊